packages/cli/src/tools/brewInstall.ts

```diff
@@ -1,16 +1,27 @@
 import {logger} from '@react-native-community/cli-tools';
 import execa from 'execa';
-import Ora from 'ora';
+import chalk from 'chalk';
+import ora from 'ora';
 
-async function brewInstall(pkg: string, loader: Ora.Ora) {
-  loader.start(`Installing ${pkg}`);
+type InstallArgs = {
+  pkg: string;
+  label?: string;
+  loader: ora.Ora;
+};
+
+async function brewInstall({pkg, label, loader}: InstallArgs) {
+  loader.start(label);
   try {
     await execa('brew', ['install', pkg]);
+
     loader.succeed();
   } catch (error) {
-    logger.log(error.stderr);
-    loader.fail(
-      `An error occured while trying to install ${pkg}. Please try again manually: brew install ${pkg}`,
+    loader.fail();
+    logger.log(`\n${error.stderr}`);
+    logger.log(
```

Comment:
shouldn't we use `logger.error` here?

Reply:
I forgot that there's a `logger.error`, this is how it looks with it:

![image](https://user-images.githubusercontent.com/6207220/64616307-9fdf7b00-d3dc-11e9-91fd-443d6ab9fc39.png)

I would rather keep the `logger.log` because the `error` prefix here doesn't really add much, same goes for `Error: Command failed: brew install watchman` as it's already mentioned below.

Reply:
Ok, so let's maybe dim the output and highlight (chalk.bold) the brew install command

Reply:
![image](https://user-images.githubusercontent.com/6207220/64616726-6a875d00-d3dd-11e9-8079-aaa9cf366b0a.png)

Like this? I like it.

Reply:
👍

Reply:
Cool, just pushed the change!